src/components/AppProvider/utilities/createAppProviderContext/tests/createAppProviderContext.test.tsx

```diff
@@ -1,7 +1,8 @@
 import * as React from 'react';
 import * as appBridge from '@shopify/app-bridge';
 import {noop} from '@shopify/javascript-utilities/other';
-import * as targets from '@shopify/react-utilities/target';
+
+import * as targets from '../../../../../utilities/target';
```

Comment:
`import * as ....` is [on its way out to enable compiling TS with babel directly](https://github.com/Shopify/web/pull/13571).

I know this isn't strictly in the scope of this PR but would you be able to look at working out how we can avoid doing  an `import * as targets` for the tests that mock the result of isServer?